certbot/certbot/_internal/tests/crypto_util_test.py

```diff
@@ -199,8 +205,7 @@ def test_bad_key_type(self):
         # Try a bad --key-type
         with pytest.raises(errors.Error,
                            match=re.escape('Invalid key_type specified: unf.  Use [rsa|ecdsa]')):
-            OpenSSL.crypto.load_privatekey(
-                OpenSSL.crypto.FILETYPE_PEM, make_key(2048, key_type='unf'))
+            make_key(2048, key_type='unf')
```

Comment:
Just explicitly noting for posterity that although this is a change from what the previous version did, it still performs the necessary test and imo is better than the previous version.